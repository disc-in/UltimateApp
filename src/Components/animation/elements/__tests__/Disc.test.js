import React from 'react';
import { render } from '@testing-library/react-native';

import Disc from '../Disc';

describe('<Disc />', () => {
  it('renders correctly', () => {
    const { toJSON } = render(<Disc baseWidth={10} number={1} />);
    expect(toJSON()).toMatchSnapshot();
  });
});
