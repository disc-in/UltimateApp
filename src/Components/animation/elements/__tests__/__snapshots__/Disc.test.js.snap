// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Disc /> renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderColor": "black",
        "justifyContent": "center",
        "textAlign": "center",
        "textAlignVertical": "center",
      },
      Object {
        "borderRadius": 6.666666666666667,
        "borderWidth": 0.6666666666666667,
        "height": 6.666666666666667,
        "width": 6.666666666666667,
      },
    ]
  }
/>
`;
